### ISSUE TITLE
Article: Learning Rust in 2024. 2 exercise listed as medium are hard.

### ISSUE THREAD

**Issue by Afsoon** [pretzelhammer/rust-blog#92]
Hello,

I'm following you article Learning Rust in 2024, and I have notice a small mistake. When you suggesting medium difficult exercises, there two that are wrong:

- Macros
- Parallel Letter Frecuency

If you follow the link, you would notice they are classified as hard instead of medium at Exercism.

**Comment by pretzelhammer** [comment]
good catch, when I wrote the article those were classified as medium but they must have re-classified them as hard recently, I've moved them [in this commit](https://github.com/pretzelhammer/rust-blog/commit/a87719d0d44dae9b3256291ac19be5457fabb6e3)

thanks!
